csaf_2.1/comment_resolution/log_from_csd01pr.md

```diff
@@ -16,47 +16,47 @@ The following abbreviations are used:
 | [21](https://groups.oasis-open.org/discussion/csdpr01-feedback-1064-from-tc-member) | 2025-06-17 | TS | [Invalid references](https://github.com/oasis-tcs/csaf/issues/1064) | 2025-07-18 | [Fix invalid references](https://github.com/oasis-tcs/csaf/pull/1082) | Completed |
-| [22](https://groups.oasis-open.org/discussion/csdpr01-feedback-1065-from-tc-member) | 2025-06-17 | TS | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) |  |  |  |
+| [22](https://groups.oasis-open.org/discussion/csdpr01-feedback-1065-from-tc-member) | 2025-06-17 | TS | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) | 2025-08-27 | [`sec` instead of section number](https://github.com/oasis-tcs/csaf/issues/1065) | Completed |
 | [23](https://groups.oasis-open.org/discussion/discussion-of-csaf-v21-from-an-end-user) | 2025-06-17 | Michael Hoover | [Add remediation `category` priority](https://github.com/oasis-tcs/csaf/issues/1067) | 2025-08-20 | [Comment Resolution CSD01PR: #1067](https://github.com/oasis-tcs/csaf/pull/1139) | Completed |
 | [24](https://groups.oasis-open.org/discussion/discussion-of-csaf-v21-from-an-end-user) | 2025-06-17 | Michael Hoover | [Integrate more BOMs](https://github.com/oasis-tcs/csaf/issues/1068) | 2025-08-20 | [Discussed in TC meeting, no change for CSAF v2.1, track for future consideration](https://github.com/oasis-tcs/csaf/issues/1068#issuecomment-3214677374) | No change required |
 | [25](https://groups.oasis-open.org/discussion/minor-typo) | 2025-06-17 | Art Manion | [Typo in CSDPR01 3.1.11.2 Version Type - Semantic versioning](https://github.com/oasis-tcs/csaf/issues/1066) | 2025-07-18 | [Fix typo in version type semantic versioning](https://github.com/oasis-tcs/csaf/pull/1080) | Completed |
-| [26](https://groups.oasis-open.org/discussion/csdpr01-feedback-1069-from-tc-member) | 2025-06-17 | TS | [`cite` instead of reference](https://github.com/oasis-tcs/csaf/issues/1069) |  |  |  |
-| [27](https://groups.oasis-open.org/discussion/csdpr01-feeback-1070) | 2025-06-18 | Nick Leali | [CVSS v4.0 schema updates](https://github.com/oasis-tcs/csaf/issues/1070) |  |  |  |
-| [28](https://groups.oasis-open.org/discussion/some-typos-and-recommendations-for-readability) | 2025-06-20 | Matthew Stradling | [Section 6.2.6 - Contradicting Product Status](https://github.com/oasis-tcs/csaf/issues/1122) |  |  |  |
+| [26](https://groups.oasis-open.org/discussion/csdpr01-feedback-1069-from-tc-member) | 2025-06-17 | TS | [`cite` instead of reference](https://github.com/oasis-tcs/csaf/issues/1069) | 2025-08-27 | [`cite` instead of reference](https://github.com/oasis-tcs/csaf/issues/1069)  | Completed |
```

Comment:
```suggestion
| [26](https://groups.oasis-open.org/discussion/csdpr01-feedback-1069-from-tc-member) | 2025-06-17 | TS | [`cite` instead of reference](https://github.com/oasis-tcs/csaf/issues/1069) | 2025-08-27 | Discussed in TC meeting and resolved with [Comment Resolution CSD01PR: #1069](https://github.com/oasis-tcs/csaf/pull/1167)  | Completed |
```